plugins/org.python.pydev/src/org/python/pydev/ui/dialogs/PyDialogHelpers.java

```diff
@@ -161,6 +175,65 @@ public static int openQuestionConfigureInterpreter(AbstractInterpreterManager m)
         return INTERPRETER_CANCEL_CONFIG;
     }
 
+    /**
+     * <p>Statically open a list dialog to choose an interpreter from Conda.</p>
+     * <p>It returns both the interpreter name and executable.</p>
+     * 
+     * @return NameAndExecutable
+     */
+    public static NameAndExecutable openCondaInterpreterSelection() {
+        List<File> envs = CondaPackageManager.listCondaEnvironments(PyDevCondaPreferences.getExecutable());
+        List<NameAndExecutable> treatedEnvs = null;
+
+        if (PlatformUtils.isWindowsPlatform()) {
+            treatedEnvs = getTreatedEnvs(envs, "/python.exe");
+        } else {
+            treatedEnvs = getTreatedEnvs(envs, "/bin/python");
```

Comment:
If the treatedEnv size == 0, it should show a dialog saying that it wasn't possible to find any env and bail out.